management/server/account.go

```diff
@@ -512,13 +512,11 @@ func (a *Account) getUserGroups(userID string) ([]string, error) {
 func (a *Account) getPeerDNSManagementStatus(peerID string) bool {
 	peerGroups := a.getPeerGroups(peerID)
 	enabled := true
-	if a.DNSSettings != nil {
-		for _, groupID := range a.DNSSettings.DisabledManagementGroups {
-			_, found := peerGroups[groupID]
-			if found {
-				enabled = false
-				break
-			}
+	for _, groupID := range a.DNSSettings.DisabledManagementGroups {
+		_, found := peerGroups[groupID]
+		if found {
```

Comment:
could be this `if _, found := peerGroups[groupID]; found {`

Reply:
What is the difference?

Reply:
just cosmetic based on my understanding, feel free to ignore

Reply:
ok :)
For my eyes fit better this but I try to flexible in the future.